changelog/1.20.0.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "1.20.0"
+description: "Released on 06/16/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.20.0.
+
+### Features ✨
+
+- web: A site-wide workspace template policy may now be managed at **Manage >
+  Admin > Templates > Template Policy**. If not set, a sensible default is
+  provided.
+- web: The _nodeSelectors_ field was added to workspace templates
+- other: There is a new JsonSchema for writing Coder workspaces as code template
+  with intellisense
+- web: Added a service message that's displayed to all users of the system. The
+  message may be dismissed by each user at any point and is not shown again
+  until there is a new message.
+- web: System banners can now wrap text
+- infra: TLS management (/_TODO_/)
+- web: Opt-in for HTTP Strict Transport Security was added. This setting can be
+  managed at **Manage > Admin > Infrastructure > HTTP Strict Transport
+  Security**
```

Comment:
Needs documentation

CC @Emyrk

Reply:
So I was made aware this morning that `dashboard` is not using the same code path in `coderd`, meaning my HSTS header is only partially applied. It is not applied to static files. Given how HSTS works, this should still be ok, but I can see how a customer might be confused.

Do we want to make this note anywhere? Maybe if I add the docs, I can add the note there?

Reply:
Do we have a ticket describing the work that is needed to fill in the gap(s) ?

Reply:
No, I need to have a longer conversation with Dean and maybe Kyle to figure that out.